### PR TITLE
Add api functional test

### PR DESCRIPTION
**Pull request by melnikovi**
Web API test for endpoint.

## Description

Web API test that creates shopping cart with configurable product, shipping and payment and asserts that endpoint returns required data.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: dev/tests/api-functional/testsuite/Magento/AggregatedServices/Api/Service/GetGuestCartTest.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AggregatedServices\Api;
+
+class GetGuestCartTest extends \Magento\TestFramework\TestCase\WebapiAbstract
+{
+    const SERVICE_NAME = 'aggregatedServicesGuestAggregatedCarts';
```
Comment (melnikovi) [inline dev/tests/api-functional/testsuite/Magento/AggregatedServices/Api/Service/GetGuestCartTest.php]: I don't think it's correct, will double check.

Comment (paliarush) [reply dev/tests/api-functional/testsuite/Magento/AggregatedServices/Api/Service/GetGuestCartTest.php]: The test on CI will fail if it's incorrect.

File: dev/tests/api-functional/testsuite/Magento/AggregatedServices/Api/Service/GetGuestCartTest.php [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        ];
+
+        $actualCartDetails = $this->_webApiCall($serviceInfo, ['cartId' => $cartId]);
+        $expectedCartDetails = $this->getExpectedCartDetails($cartId);
+
+        $this->assertEquals($expectedCartDetails, $actualCartDetails);
+    }
+
+    /**
+     * @param $cartId
```
Comment (paliarush) [inline dev/tests/api-functional/testsuite/Magento/AggregatedServices/Api/Service/GetGuestCartTest.php]: Param type is missing here and in other places.

**Review by masodu — COMMENTED** [review]
@melnikovi, @paliarush  Can we stick to the copyright headers that we already have for all the existing files?

**Comment by melnikovi** [comment]
@masodu yes, I will fix copyrights.

**Comment by melnikovi** [comment]
@masodu, @paliarush I resolved your comments

**Review by masodu — APPROVED** [review]
LGTM

**Pull request merged**